tests/ft_locales/ft_locales.cpp

```diff
@@ -3951,6 +3956,8 @@ void Ft_Locales::testDifferentStrengthComparison_data()
     QTest::addColumn<QString>("string2");
     QTest::addColumn<QList<MLocale::Comparison> >("comparisonExpectedResults");
 
+    // not spotting anything for pinyinsearch in icu. Quite a special cased collation.
+#ifdef ALSO_VERIFY_ICU_DOES_ITS_JOB_AS_WE_EXPECT
     QTest::newRow("zh_CN@collation=pinyinsearch")
```

Comment:
I wonder if this should just be `pinyin` instead of `pinyinsearch`. However I don't have better knowledge about it. It's a bit of a mystery where this name comes from.

Reply:
Pinyin would be something that can be expected more as collation but then again i'd expect mike fabian had some reason to use a search variant here. But all in all lot of the things here test icu more than the mlocale wrapper for it The testing here has gotten quite much out of hands.